airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -14,7 +14,7 @@ data:
   registries:
     cloud:
       enabled: true
-      dockerImageTag: 1.3.1
+      dockerImageTag: 1.3.2
```

Comment:
@arsenlosenko unless you want to pin hubspot to a previous version we don't need this tag, it will keep cloud up to date with oss